### DISCUSSION TITLE
about Interesting stars list

### DISCUSSION THREAD

**Discussion by 3domennotry** [Stellarium/stellarium#3805] [Ideas]
Hello.
I'm Japanese.
(Sorry, I can not speak English very well.)

I use Stellarium customized for my own use.

(install dir)/skycultures/myself/star_names.fab

I defined about 50 stars.
(ex. Sirius, Canopus...)

"Lesser known star names in Japan" are not included in my list.
(ex. Zubenelgenubi, Sheliak...)
![20240711_1](https://github.com/Stellarium/stellarium/assets/173982253/dee93a22-3863-42b0-9523-5995a55a946c)
![20240711_2](https://github.com/Stellarium/stellarium/assets/173982253/5030f974-ed90-4a2c-bed8-fff54373f966)

I found "Lesser known star names in Japan" in two list "Interesting double stars" and "Interesting variable stars" in search window.
![20240711_3](https://github.com/Stellarium/stellarium/assets/173982253/d4c5c377-73f5-4363-b973-183bcbf50e1f)

I selected Zubenelgenubi in the list, Nobody star selected in the sky.
Of course, this is a natural result.
Because, I do not defined this star's name in star_names.fab.

I traced source.

src/core/modules/StarMgr.cpp
line 2067 to 2103
direct codeing

I do not customized two list("Interesting double stars" and "Interesting variable stars" in search window).

This is my personal opinion.
I hope these lists to be customizable by me.
(ex. (userdir)/data/dstar.json or (install dir)/skycultures/XXXX/dstar.fab)

thank you.

**Comment by alex-w** [comment]
I've converted these lists to use designations for stars (not names)

**Comment by 3domennotry** [comment]
Always thank you.

I found that &gamma; Cas duplicate in variableStars list.

Someday someone may request that variableStars list include R Hya and &chi; Cyg.
As same as, other someone may request that doubleStars list include &alpha; Cen and &alpha; Cru.

I think that...
If possible, these lists should be placed outside of src(exe, dll).

thank you.

**Comment by 3domennotry** [comment]
I have confirmed that these lists.

To my delight, R Hya and &chi; Cyg were added to the variableStars list and &alpha; Cru to the doubleStars list.
(&alpha; Cen were added to the variableStars insted of doubleStars list.)

However, I selected 16 Cyg or &alpha; Cen A in each list, Nobody star selected in the sky.
Because
16 Cyg is defined by two stars, 16 Cyg A and 16 Cyg B.
&alpha; Cen is defined by two stars(excluding proxima), &alpha; Cen 1 and &alpha; Cen 2.

Thank you very much.

**Comment by 3domennotry** [comment]
Sorry for the delay in confirming.
I think that all problem was fixed.

Thank you very much.
